### ISSUE TITLE
[azure-arm-iothub] Cannot find namespace 'moment'

### ISSUE THREAD

**Issue by formulahendry** [Azure/azure-sdk-for-node#2271]
In TS, import the typings (`"azure-arm-iothub": "1.0.1-preview"`):
```javascript
import { IotHubDescription } from "../node_modules/azure-arm-iothub/lib/models";
```
When compiling, meet with below errors:
```bash
node_modules/azure-arm-iothub/lib/models/index.d.ts(345,21): error TS2503: Cannot find namespace 'moment'.
```

**Comment by amarzavery** [comment]
@formulahendry Thanks for reporting g the issue. We shall fix this in the upcoming release. To unblock yourself please add `import * as moment from "moment"` in the index.d.ts file under the models folder. mostly you should have moment installed under the node_modules folder. if not then install moment. npm install moment.

**Comment by formulahendry** [comment]
Unfortunately, I use it in CI, so I have no chance to add this. In short term, to make CI pass, it seems that I need to remove the "../node_modules/azure-arm-iothub/lib/models" from TS file, Do you know if there is any other workaround? After that, I will just wait for this to be fixed. 😃

**Comment by amarzavery** [comment]
I see. will publish an updated version today that fixes this problem.

**Comment by amarzavery** [comment]
@formulahendry - We have published a new version of azure-arm-iothub@1.1.0-preview with the fix and other updates.
https://www.npmjs.com/package/azure-arm-iothub

**Comment by formulahendry** [comment]
Works like a charm. Thanks for the quick fix!
